posts/2023-09-26-23.0.0.10-beta.adoc

```diff
@@ -0,0 +1,635 @@
+
+| mp.metrics.distribution.histogram.min-value
+| When percentile-histogram is enabled for a Timer, this property defines a lower bound for the buckets reported. Accepts a single integer or decimal value for a metric name pairing.
+
+|mp.metrics.distribution.timer.max-value
+| When percentile-histogram is enabled for a Histogram, this property defines a upper bound for the buckets reported. Accepts a single decimal values with a time unit appended (i.e., ms, s, m, h) for a metric name pairing. Accepts for a single decimal value with a time unit appended (i.e., ms, s, m, h) for a metric name pairing.
+
+|mp.metrics.distribution.timer.min-value
+| When percentile-histogram is enabled for a Histogram, this property defines a lower bound for the buckets reported. Accepts a single decimal value with a time unit appended (i.e., ms, s, m, h) for a metric name pairing.
+|===
+
+
+For example, the `mp.metrics.distribution.percentiles` can be defined as:
+
+[source]
+----
+mp.metrics.distribution.percentiles=alpha.timer=0.5,0.7,0.75,0.8;alpha.histogram=0.8,0.85,0.9,0.99;delta.*=
+----
+
+This example creates the `alpha.timer` timer metric to track and output the 50th, 70th, 75th, and 80th percentile values. The `alpha.histogram` histogram metric outputs the 80th, 85th, 90th, and 99th percentiles values. Any Histogram or Timer metric that matches with `delta.*` has its percentiles disabled.
+
+We'll expand on the above example and define histogram buckets for the `alpha.timer` timer metric using the `mp.metrics.distribution.timer.buckets` property:
+
+[source,xml]
+----
+mp.metrics.distribution.timer.buckets=alpha.timer=100ms,200ms,1s
+----
+
+This configuration tells the metrics runtime to track and output the count of durations that fall within 0-100ms, 0-200ms, and 0-1 seconds. This is due to the  histogram buckets working in a _cumulative_ fashion.
+
+The corresponding Prometheus output for the `alpha.timer` metric at the `/metrics` REST endpoint is:
+
+[source]
+----
+# HELP alpha_timer_seconds_max  
+# TYPE alpha_timer_seconds_max gauge
+alpha_timer_seconds_max{scope="application",} 5.633
+# HELP alpha_timer_seconds  
+# TYPE alpha_timer_seconds histogram <1>
+alpha_timer_seconds{scope="application",quantile="0.5",} 0.67108864
+alpha_timer_seconds{scope="application",quantile="0.7",} 5.603590144
+alpha_timer_seconds{scope="application",quantile="0.75",} 5.603590144
+alpha_timer_seconds{scope="application",quantile="0.8",} 5.603590144
+alpha_timer_seconds_bucket{scope="application",le="0.1",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="0.2",} 0.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="1.0",} 1.0 <2>
+alpha_timer_seconds_bucket{scope="application",le="+Inf",} 2.0  <2><3>
+alpha_timer_seconds_count{scope="application",} 2.0
+alpha_timer_seconds_sum{scope="application",} 6.333
+----
+
+* The Prometheus metric type is `histogram`. Both the quantiles or percentiles and buckets are represented under this type. 

```

Comment:
These should have the <1> , <2> , <3> which correspond to the <1> <2> <3> annotations in the example above.